manifest/dev.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+
+  <remote fetch="https://github.com/couchbase/" name="couchbase"/>
+  <remote fetch="https://github.com/couchbaselabs/" name="couchbaselabs"/>
+  <remote fetch="https://github.com/couchbasedeps/" name="couchbasedeps"/>
+  <remote fetch="https://github.com/elazarl/" name="elazarl"/>
+  <remote fetch="https://github.com/gorilla/" name="gorilla"/>
+  <remote fetch="https://github.com/natefinch/" name="natefinch"/>
+  <remote fetch="https://github.com/rcrowley/" name="rcrowley"/>
+  <remote fetch="https://github.com/samuel/" name="samuel"/>
+  <remote fetch="https://github.com/snej/" name="snej"/>
+  <remote fetch="https://github.com/tleyden/" name="tleyden"/>
+  <remote fetch="https://github.com/mreiferson/" name="mreiferson"/>
+  <remote fetch="https://github.com/kardianos/" name="kardianos"/>
+  <remote fetch="https://github.com/coreos/" name="coreos"/>
+  <remote fetch="https://github.com/jonboulle/" name="jonboulle"/>
+  <remote fetch="https://github.com/satori/" name="satori"/>
+  <remote fetch="https://github.com/brett19/" name="brett19"/>
+
+  <remote fetch="ssh://git@github.com/couchbaselabs/" name="couchbaselabs_private" />
+  
+  <default remote="couchbase" revision="master"/>
+
+  <!-- Build Scripts (required on CI servers) -->
+  <project name="build" path="cbbuild" remote="couchbase">
+    <annotation name="VERSION" value="0.0.0"     keep="true"/>
+    <annotation name="BLD_NUM" value="@BLD_NUM@" keep="true"/>
+    <annotation name="RELEASE" value="@RELEASE@" keep="true"/>
+  </project>
+
+  
+  <!-- Sync Gateway -->
+  <project name="sync_gateway" path="godeps/src/github.com/couchbase/sync_gateway" remote="couchbase" revision="release/1.5.0"/> 
```

Comment:
since it's a dev build, I'd expect this to a feature branch

Reply:
This particular toy build is just trying to build w/ a dependency change.